woocommerce/woocommerce-android
# Remove QR code scanning CTA from login flow

Closes: #12301 


### Description
Currently, the QR code generation is broken. The backend fix is not going to happen in the short term, so for now:
- we'll update the Woo login email template to remove the QR code entirely
- disable any mentions to QR code login from the Woo app.

<img width="1418" alt="Screenshot 2024-08-12 at 09 44 54" src="https://github.com/user-attachments/assets/8ed74d80-d743-43de-8b7e-4eff47ed09f3">

### Testing information
1.  Start login flow
2. Enter a WP.con / Jetpack connected site url
3. Enter WP.com email
4. Click on `Log in with magic link`
5. Check the button `Scan QR code to login` button is gone

| Before  | After |
| ------------- | ------------- |
| <img width="300"  src="https://github.com/user-attachments/assets/75bc5bf7-ceb9-4b85-8f4b-49d146982af4">  | <img width="300"  src="https://github.com/user-attachments/assets/fdbba6fe-a53f-453c-a10f-29d1574c74b4">  | Cell  | Cell  |  


## Images/gif



Small UI change and mostly removing code. Considered unit tests where unnecessary for this.
